gatsby-browser.js

```diff
@@ -5,4 +5,5 @@
  */
 
 // You can delete this file if you're not using it
-import 'bootstrap/dist/css/bootstrap.css';
\ No newline at end of file
+import 'bootstrap/dist/css/bootstrap.css'
+import './src/components/bootstrap-overrides.css'
```

Comment:
these overrides get imported after bootstrap so that the styling will get applied later